juego.py

```diff
@@ -484,9 +496,11 @@ def game_over(self, score, high_score):
 
                 elif event.type == MOUSEBUTTONDOWN:
                     if event.button == 1:
-                        if event.pos[0] > sx(470) and event.pos[0] < sx(470) + \
+                        if event.pos[0] > sx(470) and \
+                                event.pos[0] < sx(470) + \
                                 quit.get_width() and \
-                                event.pos[1] > sy(700) and event.pos[1] < sy(700) + \
+                                event.pos[1] > sy(700) and \
+                                event.pos[1] < sy(700) + \
```

Comment:
For readability and pep8 standard, the operators should be on the left of the operands.

Reply:
Sure, will fix it.